examples/shared/src/main/scala/zio/examples/LayerDefinitionExample.scala

```diff
@@ -17,12 +17,12 @@ object LayerDefinitionExample extends App {
   }
 
   override def run(args: List[String]): URIO[ZEnv, ExitCode] = {
-    val liveLayer: ULayer[Has[Foo]] =
-      (Console.live ++ ZLayer.succeed("Hello") ++ ZLayer.succeed(3)) >>> Foo.live
+    val liveLayer: URLayer[Has[String], Has[Foo]] =
```

Comment:
Maybe better to leave the example as is and add a test to verify that this operator is working correctly?

Reply:
That's a much better idea :)

Reply:
So yeah. Very strangely, it only compiles for Scala 3 if given a type signature. Perhaps I can turn this into a bug report. 😞